### PR TITLE
chore(chrome): update prop descriptions

### PR DESCRIPTION
**Pull request by KICKdesigns**
## Description

* Standardize prop descriptions.

## Checklist

- [ ] :ok_hand: design updates are Garden Designer approved (add the
      designer as a reviewer)
- [ ] :globe_with_meridians: Styleguidist demo is up-to-date (`yarn start`)
- [ ] :arrow_left: renders as expected with reversed (RTL) direction
- [ ] :metal: renders as expected with Bedrock CSS (`?bedrock`)
- [ ] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [ ] :guardsman: includes new unit tests
- [ ] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

**Review by vsrixyz — APPROVED** [review]
Looks good.

File: packages/chrome/src/elements/header/HeaderItem.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ interface IHeadItemProps
   extends IStyledBaseHeaderItemProps,
     IStyledLogoHeaderItemProps,
     HTMLAttributes<HTMLElement> {
+  /** Determines if the header contains a logo and prepares the header content for the logo space */
```
Comment (jzempel) [inline packages/chrome/src/elements/header/HeaderItem.tsx:22]: Please simplify as the latter portion is not accurate:

> Determines if the header contains a logo ~and prepares the header content for the logo space~

File: packages/chrome/src/elements/nav/NavItem.tsx [hunk 2]
```diff
@@ -13,21 +13,13 @@ import { useNavContext } from '../../utils/useNavContext';
...
   product?: PRODUCT;
-  /**
-   * Indicate which item is current in the nav
-   **/
+  /** Determines if the current item is in the navigation */
   isCurrent?: boolean;
-  /**
-   * Indicate that the item contains a product logo
-   */
+  /** Determines if the item contains a product logo and prepares the item content to display the logo */
```
Comment (jzempel) [inline packages/chrome/src/elements/nav/NavItem.tsx:20]: Determines if the item contains a product logo ~and prepares the item content to display the logo~

File: packages/chrome/src/elements/nav/NavItem.tsx [hunk 3]
```diff
@@ -13,21 +13,13 @@ import { useNavContext } from '../../utils/useNavContext';
...
   hasLogo?: boolean;
-  /**
-   * Indicate that the item contains the company brandmark
-   */
+  /** Determines if the item contains a brandmark and prepares the item content to display the brandmark */
```
Comment (jzempel) [inline packages/chrome/src/elements/nav/NavItem.tsx:22]: Determines if the item contains a brandmark ~and prepares the item content to display the brandmark~

File: packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx [hunk 4]
```diff
@@ -18,8 +18,15 @@ import {
...
 export interface ICollapsibleSubNavItemProps
   extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'onChange'> {
+  /** Sets the header */
   header?: React.ReactNode;
+  /** Determines if the item is expanded */
   isExpanded?: boolean;
+  /**
+   * Handles item expansion changes
+   *
+   * @param {boolean} isExpanded Determines if the item is expanded
```
Comment (jzempel) [inline packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx:28]: The parameter is an indicator of expansion state:

> ~Determines~ Indicates if the item is expanded

File: packages/chrome/src/elements/subnav/SubNavItem.tsx [hunk 5]
```diff
@@ -11,6 +11,7 @@ import { StyledSubNavItem } from '../../styled';
 import { useChromeContext } from '../../utils/useChromeContext';
 
 interface ISubNavItemProps {
+  /** Determines if the current item is in the sub-navigation */
```
Comment (jzempel) [inline packages/chrome/src/elements/subnav/SubNavItem.tsx:14]: It looks like we chose to use the term "subnav" here: https://garden.zendesk.com/components/chrome#navigation-panel. Let's remain consistent and replace "sub-navigation" with "subnav".

**Review by jzempel — CHANGES_REQUESTED** [review]

File: packages/chrome/src/elements/header/HeaderItem.tsx (hunk 1 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/header/HeaderItem.tsx]: Fixed in latest update.

File: packages/chrome/src/elements/nav/NavItem.tsx (hunk 2 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/nav/NavItem.tsx]: Fixed in latest update.

File: packages/chrome/src/elements/nav/NavItem.tsx (hunk 3 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/nav/NavItem.tsx]: Fixed in latest update.

File: packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx (hunk 4 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx]: Fixed in latest update. Should we use Indicates (in general) for all boolean descriptions?

File: packages/chrome/src/elements/subnav/SubNavItem.tsx (hunk 5 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/subnav/SubNavItem.tsx]: Fixed in latest update.

File: packages/chrome/src/elements/Chrome.tsx [hunk 6]
```diff
@@ -14,9 +14,9 @@ import { ChromeContext } from '../utils/useChromeContext';
 import { StyledChrome } from '../styled';
 
 interface IChromeProps extends HTMLAttributes<HTMLDivElement> {
-  /** Apply a custom hue to chrome navigation */
+  /** Sets a custom hue to the chrome navigation */
   hue?: string;
-  /** Prevent fixed positioning from being applied to the <html> element */
+  /** Determines if fixed positioning should not be applied to the `<html>` element */
```
Comment (jzempel) [inline packages/chrome/src/elements/Chrome.tsx:19]: Passive->Active

> Prevents fixed positioning from being applied...

**Review by jzempel — CHANGES_REQUESTED** [review]
Please double check for passive->active voice.

File: packages/chrome/src/elements/Chrome.tsx (hunk 6 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/Chrome.tsx]: Fixed in latest push

File: packages/chrome/src/elements/body/Body.tsx [hunk 7]
```diff
@@ -11,9 +11,7 @@ import { StyledBody } from '../../styled';
 import { BodyContext } from '../../utils/useBodyContext';
 
 interface IBodyProps {
-  /**
-   * Prepare the body content height to allow space for a footer component
-   **/
+  /** Applies a footer and prepares the body content height for the footer space */
```
Comment (jzempel) [inline packages/chrome/src/elements/body/Body.tsx:14]: This prop does not actually add a footer to the `Chrome`. It only "Prepares the body content height for the addition of a footer" (or something to that effect).

File: packages/chrome/src/elements/header/HeaderItem.tsx [hunk 8]
```diff
@@ -19,6 +19,7 @@ interface IHeadItemProps
   extends IStyledBaseHeaderItemProps,
     IStyledLogoHeaderItemProps,
     HTMLAttributes<HTMLElement> {
+  /** Applies a logo to the header */
   hasLogo?: boolean;
```
Comment (jzempel) [inline packages/chrome/src/elements/header/HeaderItem.tsx:23]: This prop does not add a logo to the header. Rather it indicates that the header *item* contains a product logo.

File: packages/chrome/src/elements/nav/NavItem.tsx [hunk 9]
```diff
@@ -13,21 +13,13 @@ import { useNavContext } from '../../utils/useNavContext';
...
-  /**
-   * Applies product-specific color palette
-   **/
+  /** Sets a product-specific color palette */
   product?: PRODUCT;
-  /**
-   * Indicate which item is current in the nav
-   **/
+  /** Adds the current item to the nav */
   isCurrent?: boolean;
```
Comment (jzempel) [inline packages/chrome/src/elements/nav/NavItem.tsx:19]: This prop does not add an item to the nav. It indicates which item in the nav is current. See https://garden.zendesk.com/components/chrome#default for details.

File: packages/chrome/src/elements/nav/NavItem.tsx [hunk 10]
```diff
@@ -13,21 +13,13 @@ import { useNavContext } from '../../utils/useNavContext';
...
-  /**
-   * Indicate that the item contains a product logo
-   */
+  /** Adds a product logo to the item */
   hasLogo?: boolean;
```
Comment (jzempel) [inline packages/chrome/src/elements/nav/NavItem.tsx:21]: This prop does not add a product logo to the nav item. It indicates that the item contains a product logo.

File: packages/chrome/src/elements/nav/NavItem.tsx [hunk 11]
```diff
@@ -13,21 +13,13 @@ import { useNavContext } from '../../utils/useNavContext';
...
-  /**
-   * Indicate that the item contains the company brandmark
-   */
+  /** Adds a brandmark to the item */
   hasBrandmark?: boolean;
```
Comment (jzempel) [inline packages/chrome/src/elements/nav/NavItem.tsx:23]: This prop does not add a brandmark to the nav item. Rather, it indicates that the item in question contains the company brandmark.

File: packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx [hunk 12]
```diff
@@ -18,8 +18,15 @@ import {
 
 export interface ICollapsibleSubNavItemProps
   extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'onChange'> {
+  /** Sets the header */
   header?: React.ReactNode;
+  /** Expands the nav item */
   isExpanded?: boolean;
```
Comment (jzempel) [inline packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx:24]: This prop does not perform the act of expanding the nav item. The `onChange` prop is responsible for that. Rather, it indicates whether the nav item is expanded.

File: packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx [hunk 13]
```diff
@@ -18,8 +18,15 @@ import {
 
 export interface ICollapsibleSubNavItemProps
   extends Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'onChange'> {
+  /** Sets the header */
   header?: React.ReactNode;
```
Comment (jzempel) [inline packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx:22]: It's not entirely apparent from the name, but this prop "Sets the collapsible header content". View the code under https://garden.zendesk.com/components/chrome#navigation-panel for details.

File: packages/chrome/src/elements/subnav/SubNavItem.tsx [hunk 14]
```diff
@@ -11,6 +11,7 @@ import { StyledSubNavItem } from '../../styled';
 import { useChromeContext } from '../../utils/useChromeContext';
 
 interface ISubNavItemProps {
+  /** Adds the current item to the subnav */
   isCurrent?: boolean;
```
Comment (jzempel) [inline packages/chrome/src/elements/subnav/SubNavItem.tsx:15]: This prop does not add an item to the subnav. It indicates which item in the subnav is current.

**Review by jzempel — CHANGES_REQUESTED** [review]
It looks like you dipped into the `styled` directory on this one. Please don't bother with any doc updates outside of the `elements` directory.

File: packages/chrome/src/elements/subnav/SubNavItem.tsx (hunk 14 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/subnav/SubNavItem.tsx:15]: Updated in latest push.

File: packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx (hunk 13 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx:22]: Updated in latest push.

File: packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx (hunk 12 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/subnav/CollapsibleSubNavItem.tsx:24]: Updated in latest push.

File: packages/chrome/src/elements/nav/NavItem.tsx (hunk 11 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/nav/NavItem.tsx:23]: Updated in latest push.

File: packages/chrome/src/elements/nav/NavItem.tsx (hunk 10 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/nav/NavItem.tsx:21]: Updated in latest push.

File: packages/chrome/src/elements/nav/NavItem.tsx (hunk 9 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/nav/NavItem.tsx:19]: Updated in latest push.

File: packages/chrome/src/elements/header/HeaderItem.tsx (hunk 8 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/header/HeaderItem.tsx:23]: Updated in latest push.

File: packages/chrome/src/elements/body/Body.tsx (hunk 7 above)
Comment (KICKdesigns) [reply packages/chrome/src/elements/body/Body.tsx]: Updated in latest push.

**Pull request closed without merging**